### ISSUE TITLE
Unable to display a pdf served by a protected resource

### ISSUE THREAD

**Issue by masahmad71** [react-pdf-viewer/react-pdf-viewer#203]
This can be achieved by including withCredentials parameter when calling pdfjsLib.getDocument().

**Comment by phuoc-ng** [comment]
Thanks for the suggestion.
I'll add support for this in the next version.

**Comment by phuoc-ng** [comment]
@masahmad71 @mmichael0413  

The feature was implemented and will be available in the next 2.2.0.

~~~ javascript
import { Viewer } from '@react-pdf-viewer/core';

<Viewer
    fileUrl={...}
    authorization='test'
    httpHeaders={{
        key1: 'value1',
        key2: 'value2',
    }}
/>
~~~

With the sample code above, you can see additional headers sent to server:

<img width="852" alt="Screen Shot 2020-10-31 at 22 24 47" src="https://user-images.githubusercontent.com/57786711/97783135-49c52c80-1bc8-11eb-8229-47017c0b1192.png">

The options are described in the [CHANGELOG](https://github.com/react-pdf-viewer/react-pdf-viewer/blob/master/CHANGELOG.md#v220-not-released-yet)

**Comment by phuoc-ng** [comment]
@masahmad71 v2.2.0 was [released](https://react-pdf-viewer.dev/updates/). Please upgrade.
